docs/reference/cluster/stats.asciidoc

```diff
@@ -1440,7 +1436,6 @@ The API returns the following response:
                "vm_name": "OpenJDK 64-Bit Server VM",
                "vm_version": "12+33",
                "vm_vendor": "Oracle Corporation",
-               "bundled_jdk": true,
```

Comment:
Is removing this item from this API considered a breaking change?

Reply:
I had thought the cluster stats API was not considered covered under our backcompat, but it appears it is in a bit of a gray area. I've added it back here, just always returning `true`.